command/agent/sdnotify.go

```diff
@@ -0,0 +1,9 @@
+package agent
```

Comment:
We'll need to have a copyright header at the top of this file, identical to the one on all the other files in this directory.